frontend/dist/index.html

```diff
@@ -1,11 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <script src="/bundle.js" ></script>
-  <meta charset="UTF-8">
-</head>
-
-<body>
-  <div id="mount"></div>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8" />
+  <script type="text/javascript" src="/bundle.js"></script></head>
+  <body>
+    <div id="mount"></div>
+  </body>
+</html>
```

Comment:
As you're cleaning up, the missing line breaks at the end of some files is causing a lot of unnecessary diff markers.

Reply:
i didn't make this change... trying to figure out why it's happening

Reply:
Right, I know. When there is no longer break at the end of the file, that line shows up when there is a change anywhere else in the file.